src/attributes/attr.js

```diff
@@ -34,7 +34,7 @@ jQuery.extend( {
 		}
 
 		// Fallback to prop when attributes are not supported
-		if ( typeof elem.getAttribute === "undefined" ) {
+		if ( elem.getAttribute === undefined ) {
```

Comment:
Also, pretty sure we added tests for that, so if they pass we should definitely be in the clear